protected/humhub/modules/user/messages/ru/forms_AccountChangeUsernameForm.php

```diff
@@ -0,0 +1,6 @@
+<?php
+return array (
```

Comment:
You should use short array as well as add the translations to every language in the `/message` directory.

Reply:
@Felli I think we can ignore that, because the Yii2 Message command creates such files with this syntax automatically.